app/assets/stylesheets/application/modules/_kids.scss

```diff
@@ -181,6 +181,9 @@ $kids-speech-tabs-width: 40px;
     display: flex;
     flex-direction: column;
     align-items: center;
+    #actualBoardText{
```

Comment:
What is this? And why it is `board` instead of state?

Also, aren't we using `dash-case` for selectors, and using the `mu-` prefix for classes and ids?